R/req-body.R

```diff
@@ -226,6 +226,39 @@ req_body_apply <- function(req) {
   req
 }
 
+check_json_content_type <- function(content_type, call = caller_env()) {
```

Comment:
I wonder if it's worth slightly extending `check_content_type()` for this? (i.e. make it work with both requests and responses.

Reply:
Yes, seems worth it.